rotkehlchen/db/addressbook.py

```diff
@@ -193,3 +193,32 @@ def get_addressbook_entry_name(
             result = read_cursor.fetchone()
 
         return None if result is None else result[0]
+
+    def maybe_make_entry_name_multichain(
+            self,
+            address: ChecksumEvmAddress,
+            book_type: AddressbookType = AddressbookType.PRIVATE,
+    ) -> None:
+        """Make the existing name for the specified address apply to all chains.
+        If there is no existing name or different names in different chains, then
+        no action is taken.
+        """
+        with self.write_ctx(book_type) as write_cursor:
+            if len(names := write_cursor.execute(
+                'SELECT name FROM address_book WHERE address=?',
+                (address,),
+            ).fetchall()) == 0 or len(set(names)) > 1:
+                return  # If no name, or different names on different chains, leave as is.
+
+            # else, make the name multichain
+            if len(names) == 1:  # Only one entry. Just update it.
+                write_cursor.execute(
+                    'UPDATE address_book SET blockchain=? WHERE address=?',
+                    (ANY_BLOCKCHAIN_ADDRESSBOOK_VALUE, address),
+                )
+            else:  # Same label is set on multiple chains. Delete existing entries and add new one.
+                write_cursor.execute('DELETE FROM address_book WHERE address=?', (address,))
+                write_cursor.execute(
+                    'INSERT INTO address_book(blockchain, name, address) VALUES (?, ?, ?)',
+                    (ANY_BLOCKCHAIN_ADDRESSBOOK_VALUE, names[0][0], address),
+                )
```

Comment:
it shouldn't trigger any error but just to be 1000% sure can you do insert or ignore?